chain/network/src/peer_manager/peer_manager_actor.rs

```diff
@@ -370,8 +370,13 @@ impl PeerManagerActor {
 
         let my_peer_id = config.node_id();
         let network_graph = Arc::new(RwLock::new(routing::GraphWithCache::new(my_peer_id.clone())));
-        let routing_table_addr =
-            routing::Actor::new(clock.clone(), store.clone(), network_graph.clone()).start();
+        let arbiter = Arbiter::new();
```

Comment:
as this is important place - let's add a comment that RoutingTableActor must start in a separate arbiter to prevent block of peer_manager thread.